website/blog/2022-10-07-Harness-Platform/getting_start_harness_platform.md

```diff
@@ -0,0 +1,127 @@
+
+In each Stage, you define what you want to deploy using Services, where you want to deploy it using Environments, and how you want to deploy it using Execution steps.
+You can refer to the docs below and get yourself familiar with some basics as well deployment tutorials.
+
+You can go through the following docs to get familar with the basics as well as deployment tutorials as well as sample CD pipeline for you to try out:
+
+- [CD Sample Pipeline](https://github.com/harness-community/cd-pipeline-sample)
+- [CD how to](https://docs.harness.io/category/pfzgb4tg05-howto-cd)
+
+You can go through Harness docs as well as Harness University courses to get a clarity on Harness CD:
+
+- [Harness CD Docs](https://docs.harness.io/category/pfzgb4tg05-howto-cd)
+- [Harness CD University](https://university.harness.io/page/continuous-delivery)
+
+# Feature Flags 
+
+Empower product development teams to quickly release new features, with minimal risk. Test with a specified subset, or roll out to all customers.
+
+**Feature flags** create private swim lanes for developer where they can ship the feature directly to customers and then control over who sees it , get feedback and turn on and off when needed.
+
+We recommend you to follow the below blog and implement your first feature flag , I am sure you are going to love the way it works :
+
+- [Implement your first Feature Flag](https://harness.io/blog/implement-your-first-feature-flag)
+
+There are some well curated courses as well as docs on Feature Flags which will help you get a much deeper insight about the product :
+
+- [Harness Feature Flags Docs](https://docs.harness.io/category/vjolt35atg-feature-flags)
+- [Harness Feature Flags University Course](https://university.harness.io/page/harness-feature-flags)
+
+
+# Cloud Cost Management(CCM)
+
+Harness Cloud Cost Management empowers engineers with cloud cost visibility of their apps, microservices, and clusters. The best part? Tagging isn't required.
+
+Harness provides curated docs as well as courses that can surely help you out with the concept of CCM:
+
+- [Harness CCM docs](https://docs.harness.io/category/exgoemqhji-ccm)
+- [Harness CCM University Course](https://university.harness.io/page/harness-cloud-cost-management)
+
+# Service Reliability Management
+
+Harness Service Reliability Management (SRM) module helps engineering and DevOps teams to balance feature velocity and bug fixes along with the stability and reliability needs in a production environment.
+
+You can refer to the below docs as well as some organised courses and get yourself onboard with the basics as well the usage of Service Reliability Management:
+
+- [Harness SRM Blogs](https://docs.harness.io/category/ko19u4brsv-howtos-service-reliability-management)
+
+Those who want to try out SRM , we recommend them to go through Harness SRM Practitioner exam it will help you in gaining much deeper product knowledge:
```

Comment:
```suggestion
Those who want to try out SRM, we recommend them to go through the Harness SRM Practitioner exam it will help you in gaining much deeper product knowledge:
```